crates/core/src/types/block.rs

```diff
@@ -104,6 +105,28 @@ impl BlockBody {
         );
         H256(root.into())
     }
+
+    pub fn compute_receipts_root(&self, receipts: Vec<Receipt>) -> H256 {
```

Comment:
We can move this outside the `BlockBody` `impl`

Reply:
Done!